decentraland/unity-explorer
# feat: avatar reconnection on change realm

- full reconnection flow
- assigning new rooms on reconnection

tested with /goto yemel.dcl.eth and /goto genesis

# How to test

1. Start in Genesis City with your test buddy
2. One of you use `/goto yemel.dcl.eth` chat command -> (expected): you appear in the world and don't see you test buddy and cannot chat with him
3. Your test buddy uses `/goto yemel.dcl.eth` chat command -> (expected): he appears in the same world and you see him and can chat with him
4. repeat with going back to Genesis via `/goto 0,0`
